### ISSUE TITLE
Confusing installation

### ISSUE THREAD

**Issue by xeor** [att-comdev/armada#187]
**Is this a bug report or feature request?**: feature request

**Python Version** (output of `python --version`): Python 3.6.3

**Development or Deployment Environment?**: osx against a remote k8s cluster

**Release Tag or Master**: master

**Expected Behavior**: not that much confusion, using armada

**What Actually Happened**: lot of confusion..

**How to Reproduce the Issue** (as minimally as possible):

**Any Additional Comments**:
It's not clear to me how armada should be installed and used. In the docs, there are instructions telling me to do a  `docker run -d ...`, but there are not much docs about installing the cli itself. There is a guide for using it as a helm-plugin tho.. But is that the same?

So why isn't there a `pip install armanda-helm` or similar? And why do I need to run a docker container on the side? Do it need to run at the same place as tiller?

Basically, I think the install-docs are very unclear..
If the docker container-sidekick needs to be there because of many deps, why does it need to run daemonized?

**Comment by MarshM** [comment]
Thanks for the feedback.  We're working on reviewing and updating our docs, and will pull you in at a later date to help contribute to a patchset review in GerritHub.
